guides/common/modules/proc_preparing-for-disaster-recovery-with-two-active-project-servers.adoc

```diff
@@ -0,0 +1,51 @@
+[id="preparing-for-disaster-recovery-with-two-active-project-servers"]
+= Preparing for disaster recovery with two active {ProjectServer}s
+
+Create a second {ProjectServer} by restoring a backup of your first {ProjectServer}.
+Configure both servers to operate independently in their respective data centers, but ensure that their content, such as repositories or host configuration, does not drift apart over time.
+
+.Procedure
+. Back up your {ProjectServer}.
+For more information, see xref:backing-up-{project-context}-server-and-{smart-proxy-context}_admin[].
+. Restore the backup on a system that will serve as your other {ProjectServer}.
+For more information, see xref:restoring-{project-context}-server-or-{smart-proxy-context}-from-a-backup_admin[].
++
+[NOTE]
+====
+Each server must have a distinct hostname and IP address.
+This enables you to re-register hosts if one of the servers fails.
+====
+. Ensure that content on your servers remains consistent:
+* If you want both servers to manage content synchronization and content view creation, follow these guidelines to prevent content drift:
+** Regularly synchronize repositories on both servers.
+You can use the following Ansible modules to automate repository synchronization: `{ansible-namespace}.repository_sync` and `{ansible-namespace}.sync_plan`.
+** Ensure that content views on both servers match.
+* If you want one server to manage content synchronization and content view creation, use one of these features to prevent content drift:
+** If your disaster recovery site has network access to your primary site, use Inter-Server Synchronization (ISS) to ensure your disaster recovery server synchronizes its content from the primary server.
```

Comment:
```suggestion
** If your disaster recovery site has network access to your primary site, use {ISS} (ISS) to ensure your disaster recovery server synchronizes its content from the primary server.
```
blocking: Refs https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L89 and https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L85